docs/state.md

```diff
@@ -134,7 +134,7 @@ AppStore
     'general.downloads.default-save-path': string, // default path for saving files
     'general.autohide-menu': boolean, // true if the Windows menu should be autohidden
     'general.disable-title-mode': boolean, // true if title mode should always be disabled
-    'search.default-search-engine': string, // path to the open search XML
+    'search.default-search-engine': string, // name of search engine, from js/data/searchProviders.js
```

Comment:
This will need to be set inside app.sessionStore based on the user's old selected xml engine.  And only done once, after that it should jus preserve the setting.  Also some users won't have an old xml setting if it is their first time install.